plugins/self-service/src/components/common/EmptyState.tsx

```
@@ -75,7 +83,19 @@ export const EmptyState = ({
       </Typography>
       {allowed && onSyncClick && (
         <Tooltip
-          title={syncDisabled && syncDisabledReason ? syncDisabledReason : ''}
+          title={
+            showProgressPopover ? (
+              <SyncProgressPopover entries={syncProgress} />
+            ) : syncDisabled && syncDisabledReason ? (
+              syncDisabledReason
+            ) : (
+              ''
+            )
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)